File: src/java/com/codecool/gameoflife/view/DisplayConfig.java
package com.codecool.gameoflife.view;

import com.codecool.gameoflife.model.Config;
import javafx.scene.paint.Color;

import java.awt.*;

public class DisplayConfig {

    public static final double SCREEN_WIDTH = Toolkit.getDefaultToolkit().getScreenSize().getWidth() * 0.8;
    public static final double SCREEN_HEIGHT = Toolkit.getDefaultToolkit().getScreenSize().getHeight() * 0.8;
    public static final int CELL_SIDE = (int) (SCREEN_WIDTH / Config.BOARD_SIDE * 0.5);
    public static final Color ALIVE_COLOR = Color.ORCHID;
    public static final Color DEAD_COLOR = Color.LIGHTGRAY;
}

File: src/kotlin/com/codecool/gameoflife/view/DisplayConfig.kt
package com.codecool.gameoflife.view

import com.codecool.gameoflife.model.Config
import javafx.scene.paint.Color

import java.awt.*

object DisplayConfig {

    val SCREEN_WIDTH = Toolkit.getDefaultToolkit().screenSize.getWidth() * 0.8
    val SCREEN_HEIGHT = Toolkit.getDefaultToolkit().screenSize.getHeight() * 0.8
    val CELL_SIDE = SCREEN_WIDTH / Config.BOARD_SIDE * 0.5
    val ALIVE_COLOR: Color = Color.ORCHID
    val DEAD_COLOR: Color = Color.LIGHTGRAY
}
